src/px-script/sidebar/index.js

```diff
@@ -0,0 +1,122 @@
+class Sidebar {
+    constructor (el) {
+        this._el = el;
+        this.sidebarOpen = el.classList.contains("sidebar-open");
+        this.childCount = [...this._el.querySelectorAll(".submenu")].length;
+        this.submenus = this._el.querySelectorAll(".submenu");
+        this.listItems = [...this._el.querySelectorAll("li")].length;
+        this.resizeEventMenuOpen;
+        this.resizeEventSubmenuOpen;
+        this.submenuClosed;
+
+        document.addEventListener("click", e => {
+            if (!e.target.closest(".sidebar") && this.sidebarOpen) {
+                this.closeSidebar();
+                this.hideItems();
+            } else if (!e.target.closest(".submenu") && !this.submenuClosed) {
+                this.submenuCloseAll();
+            }
+        });
+
+        if (this.listItems > 5 || this.childCount > 0) {
+            const menu = document.createElement("a");
+
+            menu.setAttribute("href", "#");
+            this.hideItems();
+            menu.classList.add("sidebar-openbtn");
+            menu.innerHTML += "<i class='material-icons'>menu</i>";
```

Comment:
Unless you plan on keeping the content of the `innerHTML` (which on this line is nothing), you dont need the `+`

```suggestion
            menu.innerHTML = "<i class='material-icons'>menu</i>";
```